libs/game/sprites.ts

```diff
@@ -68,7 +68,7 @@ namespace sprites {
     export function createProjectileFromSide(img: Image, vx: number, vy: number) {
         return createProjectile(img, vx, vy, 1);
```

Comment:
Might as well fix these now?

```suggestion
         return createProjectile(img, vx, vy, SpriteKind.Projectile);
```